build.gradle

```diff
@@ -111,6 +112,12 @@ tasks.check.dependsOn integrationTest
 
 gradlePlugin {
     testSourceSets project.sourceSets.integration
+    plugins {
```

Comment:
Ok, I thought I had reverted all of the changes there. I'll see what I can do there.